### PR TITLE
Add initial amp-reach-player sample

### PR DESCRIPTION
**Pull request by jmadler**
#252

### CODE REVIEW

**Comment by kul3r4** [comment]
LGTM but the CI is failing due to some unrelated issues, will investigate

File: src/20_Components/amp-reach-player.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  #### Introduction
+
+  Embed videos hosted on Beachfront Reach in your AMP HTML files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-vimeo/">
```
Comment (kul3r4) [inline src/20_Components/amp-reach-player.html]: amp-vimeo -> amp-reach

File: src/20_Components/amp-reach-player.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    Embed videos hosted on Beachfront Reach via the `data-embed-id` attribute.  We recommend using the responsive layout for videos.
+  -->
+    <amp-reach-player
+      data-embed-id="default"
+      layout="responsive"
+      width="560" height="315"></amp-vimeo>
```
Comment (kul3r4) [inline src/20_Components/amp-reach-player.html]: amp-vimeo -> amp-reach-player

**Comment by kul3r4** [comment]
@jmadler Please get the latest changes from master which should solve the CI problem.
I noticed 2 places in which you are closing the amp-reach-player tag using amp-vimeo.

**Comment by jmadler** [comment]
PTAL

**Comment by kul3r4** [comment]
LGTM, could you squash the commits?

**Pull request merged**